Neos.Utility.Arrays/Classes/Arrays.php

```diff
@@ -199,7 +199,7 @@ public static function array_reduce(array $array, string $function, $initial = n
      * @return mixed The value found, NULL if the path didn't exist (note there is no way to distinguish between a found NULL value and "path not found")
      * @throws \InvalidArgumentException
      */
-    public static function getValueByPath(array &$array, $path)
+    public static function getValueByPath(array $array, $path)
```

Comment:
hu scary.... btw its a bit odd that we use recursion in this helper ... shouldnt be a foreach with references be faster?

Reply:
IDK I think it went between different variants over hte years, fact is the reference seems unnecessary here and I hate references, so I removed it, probably should take that out of this change htough...